index.html

```diff
@@ -53,123 +62,126 @@ <h1 class="heading-primary2">I’m a software developer</h1>
           <a href="#"><img src="/Disabled.svg" alt="arrow"></a> 
```

Comment:
- [ ] Kindly consider using the correct image path so that your images can be visible on the page for a better user experience.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/200884187-99cde626-8a90-4eb1-a192-196b8765b3f0.png)